contracts/connectors/loantoken/LoanTokenLogicStandard.sol

```diff
@@ -61,7 +61,9 @@ contract LoanTokenLogicStandard is LoanTokenSettingsLowerAdmin {
 	 * @notice Fallback function is to react to receiving value (rBTC).
 	 * */
 	function() external {
-		revert("loan token logic - fallback not allowed");
+		// Due to contract size issue, need to keep the error message to 32 bytes length / we remove the revert function
+		// Remove revert function is fine as this fallback function is not payable, but the trade off is we cannot have the custom message for the fallback function error
+		// revert("loan token-fallback not allowed");
```

Comment:
can't we leave it like this?
```suggestion
		revert("1001");
```

Reply:
I don't think we can do this, currently, this contract size is already 24 KB exactly. So, any additional function or instruction will cause contract size issue